Resources/views/Block/ajax.html.twig

```diff
@@ -0,0 +1,18 @@
+{% block block %}
```

Comment:
A reserved keyword as block name… do you have a choice or is it the way the block bundle works?